examples/cpma-config.example.yaml

```diff
@@ -4,6 +4,7 @@ SSHCreds:
   Login: "root"
   PrivateKey: "/home/example/.ssh/key"
   Port: 22
+InsecureKey: true
```

Comment:
This should be InsecureHostKey, to comment itself. Or, perhaps, even better to keep it inline with ssh go package... InsecureIgnoreHostKey.

Reply:
Inlining it with ssh credentials may require some additional and unnecessary engineering. `InsecureHostKey` is a boolean and we are reading ssh config as `map[string]string`  + more changes in `root.cmd` because this flag doesn't have string value

Reply:
By keep it inline I meant to keep the same name. InsecureKey doesn't tell you much, unlike InsecureHostKey.